e-library/src/Components/homepage/body.jsx

```diff
@@ -12,11 +15,14 @@ const Body = () => {
       {book.map((item) => (
         <div className="img">
           <img src={item.imageUrl} alt="" />
-          <button type="button">BORROW</button>
+          <Link to="/addImage" onClick={() => { bookId.id = item.id; }}>
```

Comment:
why are you mutating a global variable. this is `anti-pattern`. refactor this implementation.